docs/source/llms/tracing/index.rst

```diff
@@ -467,6 +467,17 @@ filter strings, and other parameters.
         experiment_ids=["1", "2"], filter_string="attributes.status = 'OK'", max_results=5
     )
 
+Alternatively, you can use `fluent API <./index.html#tracing-fluent-apis>`_ to search for traces. This method allows you to specify fields to extract from traces using the format 
+``"span_name.[inputs|outputs]"`` or ``"span_name.[inputs|outputs].field_name"``.
+
+.. code-block:: python
+
+    # Search for traces in specific experiments
+    traces = mlflow.search_traces(
+        experiment_ids=["1", "2"],
+        extract_fields=["span1.inputs", "span1.outputs.c"],
+    )
```

Comment:
Can we show an example output? E.g. using a screenshot?

Reply:
do we need a screenshot?
we can just copy output text of `print(traces)`